sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/json/JsonInferSchema.scala

```diff
@@ -144,6 +150,9 @@ private[sql] class JsonInferSchema(options: JSONOptions) extends Serializable {
         }
         if (options.prefersDecimal && decimalTry.isDefined) {
           decimalTry.get
+        } else if (options.inferTimestamp &&
```

Comment:
Could you adjust the comment for inferTimestamp and mention the ntz type:
https://github.com/apache/spark/blob/1235bd29f087db834777cb9fc4c53a12e7f7d6b3/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/json/JSONOptions.scala#L141

Reply:
Yes, sure. I will update, thanks for pointing it out!

Reply:
Updated! Could you check the latest PR version? Thanks.